Hoxtygen/the-refugees-stories
# chore: Setup SASS preprocessor

SASS  was setup to be used in the project.



##  What does this PR do
It setup SASS preprocessor

##  Description of task to be completed
* Install sass package from npm
* Create folder structure for different part of the app
* Test that setup works by writing out some styling.

##  How should this manually be tested
* Clone the app
* `cd` into the directory and run `npm install`
* After installation of packages  type `npm start`
* Add style rules to any of the created files and see changes

##  What are the relevant pivotal tracker stories
[Completes [#174961766](https://www.pivotaltracker.com/story/show/174961766)]

##  Screenshots (If appropriate)


##  Questions (If any):
